Examples/Tests/silver_mueller/analysis_silver_mueller.py

```diff
@@ -24,15 +25,27 @@
 
 ds = yt.load( filename )
 all_data_level_0 = ds.covering_grid(level=0,left_edge=ds.domain_left_edge, dims=ds.domain_dimensions)
-Ex = all_data_level_0['boxlib', 'Ex'].v.squeeze()
-Ey = all_data_level_0['boxlib', 'Ey'].v.squeeze()
-Ez = all_data_level_0['boxlib', 'Ez'].v.squeeze()
+warpx_used_inputs = open('./warpx_used_inputs', 'r').read()
+if re.search('geometry.dims\s*=\s*RZ', warpx_used_inputs):
+    Er = all_data_level_0['boxlib', 'Er'].v.squeeze()
+    Et = all_data_level_0['boxlib', 'Et'].v.squeeze()
+    Ez = all_data_level_0['boxlib', 'Ez'].v.squeeze()
+else:
+    Ex = all_data_level_0['boxlib', 'Ex'].v.squeeze()
+    Ey = all_data_level_0['boxlib', 'Ey'].v.squeeze()
+    Ez = all_data_level_0['boxlib', 'Ez'].v.squeeze()
 # The peak of the initial laser pulse is on the order of 6 V/m
 # Check that the amplitude after reflection is less than 0.01 V/m
 max_reflection_amplitude = 0.01
-assert np.all( abs(Ex) < max_reflection_amplitude )
-assert np.all( abs(Ey) < max_reflection_amplitude )
-assert np.all( abs(Ez) < max_reflection_amplitude )
+
+if re.search('geometry.dims\s*=\s*RZ', warpx_used_inputs):
```

Comment:
And we could reuse the boolean here:
```suggestion
if geom_RZ:
```